Notifications/notifications_actions_customization.md

```diff
@@ -12,99 +12,111 @@ This document is a starting point for engaging the community and standards bodie
+We would like to propose an extension to the [Notifications API](https://notifications.spec.whatwg.org/) standard for incoming call scenarios to allow the notification action buttons to be customized and also allow the application to play a ringtone. This capability would make the incoming call notifications, which may require a faster immediate response, clearly distinguishable from the others to the user and would also contribute to increasing accessibility on the Web.
 
 ## Goals
 
-Propose an extension to the Notifications API to allow more customization for VoIP web apps. 
+Propose an extension to the Notifications API to allow more customization for VoIP Progressive Web Apps (PWAs). 
 
 ## Non-Goals
 
 Allow general web contents to arbitrarily change action button colors and sound sources without restrictions from the underly platforms.
 
 ## Use Cases 
 
-- A video conferencing website can produce its call notification pop-up with a green "accept" button and red "decline" button.
-- Web apps will be capable to change the notification action button colors if they feel like this will improve it UX.
+- A video conferencing PWA can produce its call notification pop-up with a green "accept" button and red "decline" button.
+- Web apps will be capable to change the notification action button colors if they feel like this will improve its UX.
 
 ## Proposed Solution
 
-We propose an extension to the current Notifications API where the `NotificationAction` dictionary would have a new `style` variable of type `NotificationActionStyle`, which is an enum with 3 values: 
-- `"default"`,
-- `"success"`, and
-- `"critical"`.
-
-These styles are platform-dependent but should be distinct from each other to allow the user to easily identify its options. For example, the `"success"` action button could have a green theme - e.g., have green background color or have a green font over the default background - while the `"critical"` could be red-themed, but it is up to the platform to define the style. If the selected style is not supported by the platform, it should fallback to `"default"`
+We propose the creation of a new property `scenario` in the `NotificationOptions` dictionary of type `NotificationScenario`, that is an `enum` with two values: `"default"` and `"incoming-call"`. The color treatment and ringtone capabilities will only be available in the `"incoming-call"` scenario.
```

Comment:
Maybe we could say something about the extensibility of this enum.  Maybe something like long term, the Notifications API could support additional scenarios by adding more values to this enum like alarm, reminder....

Reply:
Added a comment about the long term extensibility of the enum. Thanks!